### PR TITLE
Make OpenStack resources cleaning script interactive

### PR DESCRIPTION
**Pull request by Xenwar**
Make OpenStack resources cleaning script interactive

Signed-off-by: Anwar Hassen <anwar.hassen@est.tech>

### CODE REVIEW

File: capo/clean-os-resources.sh [hunk 1]
```diff
@@ -1,55 +1,71 @@
...
+FILTER=${1:-"no-existing-resources"}
+datadir="/tmp/collectedsources"
 
-echo "===Removing openstack servers==="
-servers=$(openstack server list -c Name -f value | grep ${_prefix})
-for server in ${servers};do 
-  echo "Removing server: ${server}"
-  openstack server delete "${server}";
-done
+remove_resource() {
```
Comment (wgslr) [inline capo/clean-os-resources.sh:6]:
```suggestion
run_after_confirmation() {
```
Removing resources depends on the provided command, there is nothing inherently deletion-related in this function

File: capo/clean-os-resources.sh [hunk 2]
```diff
@@ -1,55 +1,71 @@
...
+    _command="${1}"
+    _message="${2}"
+    printf "\n\t${_message}\n"
+    read -p "Continue (y/n)?" DRY_RUN
```
Comment (wgslr) [inline capo/clean-os-resources.sh:10]: Why `DRY_RUN`? Maybe `USER_RESPONSE`?

File: capo/clean-os-resources.sh [hunk 3]
```diff
@@ -1,55 +1,71 @@
 #!/bin/bash
 
-_prefix=${1:-"cluster-e2e"}
+FILTER=${1:-"no-existing-resources"}
+datadir="/tmp/collectedsources"
```
Comment (wgslr) [inline capo/clean-os-resources.sh:4]:
```suggestion
datadir="/tmp/collectedresources"
```

File: capo/clean-os-resources.sh (hunk 2 above)
Comment (namnx228) [reply capo/clean-os-resources.sh:10]: Or maybe RUN_COMMAND?

Comment (wgslr) [reply capo/clean-os-resources.sh:10]: Fine with me

File: capo/clean-os-resources.sh (hunk 1 above)
Comment (Xenwar) [reply capo/clean-os-resources.sh]: done

File: capo/clean-os-resources.sh (hunk 2 above)
Comment (Xenwar) [reply capo/clean-os-resources.sh]: done

File: capo/clean-os-resources.sh (hunk 3 above)
Comment (Xenwar) [reply capo/clean-os-resources.sh]: done

**Review by wgslr — APPROVED** [review]

**Pull request merged**